python/cudf/cudf/dataframe/index.py

```diff
@@ -324,6 +324,9 @@ def is_monotonic_decreasing(self):
     def get_slice_bound(self, label, side, kind):
         raise (NotImplementedError)
 
+    def isin(self, values):
+        return self.to_series().isin(values).data.mem
```

Comment:
Even though Pandas returns a numpy array here we should return a cuDF index instead since upstream won't know what to do with a Numba device array 99% of the time